src/main/java/quickndirty/minisurveymonkey/MultipleChoiceQuestion.java

```diff
@@ -0,0 +1,32 @@
+package quickndirty.minisurveymonkey;
+
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class MultipleChoiceQuestion extends Question{
+
+    @ElementCollection
+    private List<String> choices;
+
+    public MultipleChoiceQuestion(){
+        this.type = QuestionType.MC;
+        choices = new ArrayList<>();
+    }
+
+    public MultipleChoiceQuestion(String prompt, List<String> choices) {
+        this.type = QuestionType.MC;
+        this.prompt = prompt;
+        this.choices = choices;
+    }
+
```

Comment:
Should you be able to remove a choice?

Reply:
https://github.com/austinjturner/survey-monkey-SYSC4806/pull/42/commits/8e26b72fa4dced01256c0096e10c09cd7728de51